nuxeo-wopi-web-ui/src/main/resources/web/nuxeo.war/ui/nuxeo-wopi/nuxeo-wopi.html

```diff
@@ -19,10 +19,6 @@
 
 <nuxeo-slot-content name="wopiLink" slot="BLOB_ACTIONS">
   <template>
-    <nuxeo-filter document="[[document]]" expression="!!document.contextParameters.wopi">
-      <template>
-        <nuxeo-wopi-link document="[[document]]" xpath="[[xpath]]"></nuxeo-wopi-link>
-      </template>
-    </nuxeo-filter>
+    <nuxeo-wopi-link document="[[document]]" blob="[[blob]]"></nuxeo-wopi-link>
```

Comment:
Shouldn't we keep the filter with `expression="!!blob.wopi"`?

Reply:
Unfortunately, `nuxeo-filter` does not take a blob (yet?) so you can't do `<nuxeo-filter blob="[[blob]]" expression="!!blob.wopi">`